fcrepo-import/src/main/java/org/fcrepo/importer/Importer.java

```diff
@@ -83,14 +87,10 @@ private FcrepoClient client() {
      */
     public void run() {
         logger.info("Running importer...");
-        try {
```

Comment:
OK, so they bubble up to the driver, which logs them.  Then I'm 👍  on this PR — ready to merge, @ajs6f ?